statistics/mode.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Returns the arithmetic mode(s) of the numbers in the given collection
+ *
+ * Example:
+ *
+ * ```ts
+ * import { assertEquals } from "../testing/asserts.ts"
+ * import { mode } from "./mode.ts"
+ *
+ * const numbers = [ 4, 2, 7, 4 ]
+ * assertEquals(mode(numbers), 4)
```

Comment:
Doesn't seem matching to the implementation?

Reply:
You're right, I forgot to update the example when I changed the return value to be multimodal. I'll fix that when I get off work tonight 👍

Reply:
You're right, I'll fix when I'm off work.